js/apps/admin-ui/src/components/group/GroupPickerDialog.tsx

```diff
@@ -93,14 +93,13 @@ export const GroupPickerDialog = ({
             throw new Error(t("notFound"));
           }
         }
-        if (group?.id) {
-          const args: SubGroupQuery = {
-            first: first,
-            max: max + 1,
-            parentId: group.id,
-          };
-          groups = await adminClient.groups.listSubGroups(args);
-        }
+
+        const args: SubGroupQuery = {
+          first: first,
+          max: max + 1,
```

Comment:
Without the slice below, all `max + 1` subgroups will now be shown. Maybe this should just be `max`?

Reply:
right, we still need the slice for the root groups